doc/libf3d/OPTIONS.md

```diff
@@ -94,6 +94,7 @@ ui.filename-info|string<br>-<br>render|Content of *filename info* to display.
 ui.font-file|string<br>-<br>render|Use the provided FreeType compatible font file to display text.<br>Can be useful to display non-ASCII filenames.|\-\-font-file
 ui.fps|bool<br>false<br>render|Display a *frame per second counter*.|\-\-fps
 ui.loader-progress|bool<br>false<br>load|Show a *progress bar* when loading the file.|\-\-progress
+ui.animation-progress|bool<br>false<br>load|Show a *progress bar* when playing the animation.|\-\-animation-progress
```

Comment:
`bool<br>**true**<br>load`

Reply:
Should default be specified? If so, how?

Reply:
Also, I've just notice that there is default column in user/OPTIONS. I guess I should set true in there?

Reply:
Maybe change option to --hide-animation-progress then?